ext/nnstreamer/tensor_filter/meson.build

```diff
@@ -101,20 +106,38 @@ if tflite_support_is_available
   nnstreamer_filter_tflite_deps = tflite_support_deps + [thread_dep, libdl_dep, glib_dep, gst_dep, nnstreamer_dep]
 
   tflite_compile_args = []
+  tflite_compile_args += '-DTFLITE_VERSION=1'
+  tflite_compile_args += '-DTFLITE_MINOR_VERSION=' + tflite_versions[1]
 
-  if cxx.has_type('kTfLiteInt8', prefix : '#include <tensorflow/contrib/lite/model.h>')
-    tflite_compile_args += '-DTFLITE_INT8=1'
-  endif
-  if cxx.has_type('kTfLiteInt16', prefix : '#include <tensorflow/contrib/lite/model.h>')
-    tflite_compile_args += '-DTFLITE_INT16=1'
-  endif
-  if cxx.has_type('kTfLiteFloat16', prefix : '#include <tensorflow/contrib/lite/model.h>')
-    tflite_compile_args += '-DTFLITE_FLOAT16=1'
-  endif
-  if cxx.has_type('kTfLiteComplex64', prefix : '#include <tensorflow/contrib/lite/model.h>')
-    tflite_compile_args += '-DTFLITE_COMPLEX64=1'
+  # Since tf-1.13, tflite has moved from contrib to core
+  if tflite_versions[1].to_int() >= 13
+    tflite_compile_args += '-DTFLITE_NO_CONTRIB'
+    if cxx.has_type('kTfLiteInt8', prefix : '#include <tensorflow/lite/model.h>')
```

Comment:
is it possible to set header prefix and remove duplicated lines?
```
tflite_h_prefix='#include <tensorflow/lite/model.h>'
if tflite_versions[1].to_int() < 13
tflite_h_prefix='#include <tensorflow/contrib/lite/model.h>'
endif

if cxx.has_type('kTfLiteInt8', prefix : tflite_h_prefix)
  tflite_compile_args += '-DTFLITE_INT8=1'
endif
....
```

Reply:
Yes, looks better